griptape/tools/aws_s3_client/tool.py

```diff
@@ -211,6 +212,44 @@ def upload_content_to_s3(
         except Exception as e:
             return ErrorArtifact(f"error uploading objects to the bucket: {e}")
 
+    @activity(
+        config={
+            "description": "Can be used to download an object from an AWS S3 bucket",
+            "schema": Schema(
+                {
+                    Literal(
+                        "bucket_name",
+                        description="The name of the S3 bucket to download from.",
+                    ): str,
+                    Literal(
+                        "object_key",
+                        description="The object key name to download.",
+                    ): str,
+                }
+            ),
+        }
+    )
+    def download_object(
```

Comment:
I think this should function more like `FileManager.load_files_from_disk` to provide the LLM with a way to download multiple files.

Additionally, I wonder if we should add a way to download all objects and load all files from a specific directory (and in `GoogleDriveClient.download_files`). Shouldn't be part of this PR but we should have an issue for that.